README.md

```diff
@@ -25,8 +46,43 @@ Here is a quick example tutorial on how to run the ML backend with a simple text
    label-studio-ml start my_ml_backend
    ```
    
-4. Run Label Studio connecting it to the running ML backend from the project settings page
+4. Start Label Studio and connect it to the running ML backend on the project settings page.
 
 ## Create your own ML backend
 
-Check examples in `label_studio_ml/examples` directory.
+Follow this tutorial to wrap your existing machine learning model code with the Label Studio ML SDK to use it as an ML backend with Label Studio. 
+
+Before you start, determine the following:
+1. What types of labeling does your model support? In other words, what are the expected inputs and outputs for your model? This informs the [Label Studio labeling config]() that your model supports.
+2. The [prediction format](https://labelstud.io/guide/predictions.html) returned by your ML backend server.
+
+This example tutorial outlines how to wrap a simple text classifier based on the [scikit-learn]() framework with the Label Studio ML SDK.
+
+First, create a class declaration. You can create a Label Studio-compatible ML backend server in one command by inheriting it from `LabelStudioMLBase`. 
+```python
+from label_studio_ml.model import LabelStudioMLBase
+
+class MyModel(LabelStudioMLBase):
+```
+
+Then, define loaders in the `__init__` method. 
+
+```python
+def __init__(self, **kwargs):
+    # don't forget to initialize base class...
+    super(MyModel, self).__init__(**kwargs)
+    self.model = self.load_my_model()
+```
+
+The inherited class contains fields for loading data specific to Label Studio:
+- `self.parsed_label_config` is a Python dict that provides a Label Studio project config structure. See [ref for details]()
+- `self.train_output` is a Python dict with the results of the previous model training runs. Use this if you want to load the model for the next updates for active learning and model fine-tuning.
+
+After you define the loaders, define two methods for your model:
+
+`predict(tasks, **kwargs)`, which takes [JSON-formatted Label Studio tasks](doc-link) and returns predictions in [format accepted by Label Studio](doc-link).
+`fit(annotations, **kwargs)`, which takes [JSON-formatted Label Studio annotations](doc-link) and returns an arbitrary dict where some information about the created model can be stored.
```

Comment:
do we wanna give examples of what types of info might make sense in this method or is that going to be obvious to most?

Reply:
put some examples of code along with explanations, but I guess they need to be refined a bit